app/views/document_sets/settings.html.slim

```diff
@@ -75,7 +76,7 @@
         =button_tag 'Add'
         =label_tag 'user_id', "Add Collaborator", class: 'hidden'
       -@collaborators.each do |user|
-        -contributions = user.deeds.count :conditions => "collection_id = #{@collection.slug}"
+        -contributions = user.deeds.where(collection_id: @collection.slug).count
```

Comment:
Should this be passed `@collection.slug` or `@collection.id`?  (I realize that the code is testing the same thing; I just suspect the original code was wrong (or dependent on `friendly_id`).

Reply:
@saracarl the query is basically the same, except the proposed change uses newer syntax.